index.html

```diff
@@ -0,0 +1,143 @@
+      <hr class="main-program-underline">
+      <article class="main-program-container">
+        <section class="main-program-section mp-section-1">
+          <img class="main-program-images" src="images/program_icon_01.png" alt="person standing at a podium">
+          <h3 class="secion-headings">Moivational Speeches</h3>
+          <p>Listen to the top enterpreneues from various countries
+          </p>
+        </section>
+        <section class="main-program-section">
+          <img class="main-program-images" src="images/program_icon_02.png" alt="Exhibition">
+          <h3 class="secion-headings">Exhibition</h3>
+          <p>Various new stratup opportunities for young enterpreneues</p>
+        </section>
+        <section class="main-program-section">
+          <img class="main-program-images" src="images/program_icon_03.png" alt="Forum">
+          <h3 class="secion-headings">Forum</h3>
+          <p>Have the time to share your thoughts with top enterpreneues</p>
+        </section>
+        <section class="main-program-section">
+          <img class="main-program-images" src="images/program_icon_04.png" alt="Workshop">
+          <h3 class="secion-headings">Workshop</h3>
+          <p>An opportunity to explore investment with startups</p>
+        </section>
+        <section class="main-program-section">
+          <img class="main-program-images" src="images/program_icon_05.png" alt="Learning logo">
+          <h3 class="secion-headings secion-headings-networking">Networking</h3>
+          <p>An opportunity to network with top enterprenuers from around the world</p>
+        </section>
+        <button type="submit" class="desktop-hidden main-program-button">
+          Join Startup 2022 Conference
+        </button>
+      </article>
+      <a class="main-program-bottom-text mobile-hidden" href="">See the whole program</a>
+    </section>
+
+    <section>
+      <h2 class="speaker-heading">Motivational Speakers</h2>
+      <hr class="main-program-underline speaker-underline">
+      <article id="Motivational-speakers" class="Motivational-speakers">
+      </article>
+    </section>
+    <!--------------------- The herosection------------ ------------------->
+
+  </main>
+
+  <footer class="footer-wrapper">
+    <section>
+      <div class="footer-01">
+        <span class="partner">Partner</span>
+        <hr class="main-program-underline unique-footer-underline">
+      </div>
+      <div class="footer-icons-01">
+        <img class="footer-images" src="images/background/lumen.png" alt="Icon">
+        <img class="footer-images" src="images/airbnb.png" alt="icon">
+        <img class="footer-images" src="images/wikimedia.png" alt="Icon">
+      </div>
+      <div class="footer-icons-02">
+        <img class="footer-images" src="images/background/kcc.png" alt="Icon">
+        <img class="footer-images" src="images/mozilla.png" alt="icon">
+      </div>
+    </section>
```

Comment:
# Optional:
- Please follow the design, and work again on partners section: **This section is covered with one single color and not tow colors**
- **See your design**:
![2](https://user-images.githubusercontent.com/63972670/203849011-64e5f387-0b2e-417d-830f-e63d2e5b141e.PNG)
- **Look at the Original Design**:
![1](https://user-images.githubusercontent.com/63972670/203849071-49b56b2d-58ed-4b84-a0c9-54ad15163330.PNG)